### ISSUE TITLE
`rvd` needs to check whether `openvpn_bin` exists when `connect`

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#104]
```console
$ sudo grep bin rvd.json 
  "openvpn_bin": "/opt/openvpn/bin/openvpn",
$ sudo ls -la /opt/openvpn/bin/openvpn
ls: /opt/openvpn/bin/openvpn: No such file or directory
```

```console
$ /opt/rvc/bin/rvc connect ct
name: ct
	status: CONNECTING
	openvpn-status: DISCONNECTED
	in-total: 0
	out-total: 0
	timestamp: 1505107912
```

That results in the following:
```
2017-09-11T13:31:53HKT	ERR : VPN: Couldn't connect to OpenVPN management console '127.0.0.1:6001'(err:61)(at vpn.c:224)
2017-09-11T13:31:53HKT	INFO : VPN: Failed VPN connection with name 'ct'. Try to stop OpenVPN(at vpn.c:596)
2017-09-11T13:31:53HKT	INFO : VPN: Stopping VPN connection with name 'ct'(at vpn.c:284)
```

Can you implement a check that `openvpn_bin` actually exists or not please, the log file does not currently show it.

**Comment by jjr840430** [comment]
@erikbor I see.

**Comment by jjr840430** [comment]
Fixed on #108
